pylint_nautobot/use_fields_all.py

```diff
@@ -0,0 +1,49 @@
+"""Check for CharField's or TextField's on models where null=True and blank=True."""
+from astroid import Assign
+from astroid import AssignName
+from astroid import ClassDef
+from astroid import Const
+from pylint.checkers import BaseChecker
+
+from .utils import is_version_compatible
+
+_META_CLASSES = {
+    "nautobot.core.api.serializers.NautobotModelSerializer.Meta": ">1",
+    "nautobot.core.tables.BaseTable.Meta": ">=2",
+    "nautobot.extras.filters.NautobotFilterSet.Meta": ">1",  # Capability should be added, but does not exist today
+    "nautobot.extras.forms.NautobotModelForm.Meta": ">1",
+    "nautobot.utilities.tables.BaseTable.Meta": ">1,<2",
+    # NautobotBulkEditForm - Unclear, in floor app I see it defined, but others I do not, perhaps it is an implied all, in which case I think better to be explicit and convert to respect the all param as best practices)
+    # NautobotFilterForm - Not needed (can investigate other options for field_order)
```

Comment:
Perhaps better to move the uncertain ones to a new issue and discuss there what else could be added?

Reply:
FilterForm requires a change upstream before it can be added for `__all__`

BulkEditForm I do not believe this is applicable since this does not have an ancestor of `django.forms.ModelForm`

Reply:
Comments are removed.